src/cmds/qrerun.c

```diff
@@ -132,7 +132,7 @@ main(int argc, char **argv, char **envp) /* qrerun */
 		}
 
 		stat = pbs_rerunjob(connect, job_id_out, extra);
-		if (stat && (pbs_errno != PBSE_UNKJOBID)) {
+		if ((stat && (pbs_errno != PBSE_UNKJOBID)) || (pbs_geterrmsg(connect))) {
```

Comment:
Since the exit value would now be different, I'd suggest adding this change to your EDD.